test/functional/mempool_accept_v3.py

```diff
@@ -644,4 +644,4 @@ def run_test(self):
 
 
```

Comment:
Maybe update `test/functional/wallet_create_tx.py` test.

see https://github.com/bitcoin/bitcoin/commit/d3fa531e67c0af77f52e754ad47a8396f2cab4c4, if it makes sense feel free to cherry-pick